.github/workflows/ossar-analysis.yml

```diff
@@ -44,6 +56,6 @@ jobs:
 
       # Upload results to the Security tab
     - name: Upload OSSAR results
-      uses: github/codeql-action/upload-sarif@v3
+      uses: github/codeql-action/upload-sarif@v2
```

Comment:
why we started using older action version here?

Reply:
I thought it would have to use the version 2 to work in Security Tab. I checked the CodeQL GitHub repo and they mentioned about the V3 but I didn't realize that the version could have applies to its child GitHub actions - https://github.com/github/codeql-action. I appreciate your help in leading me on the right track.